diff --git a/node_modules/@jamsch/react-native-hanzi-writer/src/HanziWriter.tsx b/node_modules/@jamsch/react-native-hanzi-writer/src/HanziWriter.tsx
index 1d95790..595455d 100644
--- a/node_modules/@jamsch/react-native-hanzi-writer/src/HanziWriter.tsx
+++ b/node_modules/@jamsch/react-native-hanzi-writer/src/HanziWriter.tsx
@@ -207,7 +207,7 @@ export function CharacterLoader({
   }
 }
 
-const TRANSFORM = 'translate(20, 248.515625) scale(0.25390625, -0.25390625)';
+const TRANSFORM = 'translate(40, 228.515625) scale(0.20390625, -0.20390625)';
 
 export type HanziWriterAnimationState =
   | 'playing'
